src/tests/Common/XUnitWrapperLibrary/TestSummary.cs

```diff
@@ -95,12 +102,66 @@ public string ToXmlString()
 
             return testResultSb.ToString();
         }
+
+        private string SanitizeOutput(string output)
+        {
+            Dictionary<char, string>? invalidChars = FindInvalidXMLChars(output);
```

Comment:
Will this do something reasonable for surrogate pairs? They tend to appear in test output because they are interesting test inputs.